.npmrc

```diff
@@ -0,0 +1 @@
+save-exact=true
```

Comment:
We have already agreed that we should do this, this just forces it in npm install without thinking about it.